website/public/js/services/groupServices.js

```diff
@@ -77,6 +77,21 @@
         $location.path("/options/groups/party");
       }
     }
+    
+    function isOverPartyCap(group) {
```

Comment:
This is client side.  Unless I'm missing something the server side function is exported as a property which will become out of date if users are invited or removed.  So as the model changes the value needs to be updated.  I'm not happy about it either though.  What I guess I would like to do is completely update `group` every time `removeMember` or `invite` is run.  This is doable with the `invite` function because it returns the model, but the `removeMember` function doesn't.